core/src/main/resources/reference.conf

```diff
@@ -93,6 +93,10 @@ akka.kafka.consumer {
   # This value is used instead of Kafka's default from `default.api.timeout.ms`
   # which is 1 minute.
   metadata-request-timeout = 5s
+
+  # Interval for cheking that transaction was completed before closing the consumer.
```

Comment:
I don't see any more documentation for the end user about his new functionality.  I suggest updating the Transactions section in the docs.

Reply:
[Minor] Typo on "checking".